src/main/scala/co/datamechanics/delight/Utils.scala

```diff
@@ -17,4 +19,20 @@ object Utils {
     bos.close()
     compressed
   }
+
+  def time[R](shouldLogDuration: Boolean, blockName: String)(block: => R): R = {
```

Comment:
nit: it could be part of `delight` [package object](https://docs.scala-lang.org/tour/package-objects.html) which would make it available to all classes "implicitly".

Reply:
Fixed in [`8b0cc1b` (#9)](https://github.com/datamechanics/delight/pull/9/commits/8b0cc1b2ebf6c96ceb27adb2ccb05d9ec9723d42)